UnitsNet/Scripts/UnitDefinitions/Length.json

```diff
@@ -70,6 +70,18 @@
       ]
     },
     {
+      "SingularName": "USSurveyFoot",
+      "PluralName": "USSurveyFeet",
+      "FromUnitToBaseFunc": "x*0.3048006096",
+      "FromBaseToUnitFunc": "x/0.3048006096",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "ftUS" ]
```

Comment:
I propose both or just "ftUS". I can't find an independent authority on the abbreviation.   The source you sited is a forum for geo software called the Manifold System which I've never heard of.   

ARCGis is more prominent software and uses ftUS. 

http://webhelp.esri.com/arcgisdesktop/9.3/index.cfm?TopicName=About_setting_distance_units